src/components/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -1,29 +1,42 @@
-import { ClipboardCopy, Split, SplitItem } from '@patternfly/react-core';
+import { ClipboardCopy, ClipboardCopyAction, Split, SplitItem } from '@patternfly/react-core';
+
+import styles from './CopyToClipboard.module.css';
 
 interface ICopyToClipboardProps {
   prefixComponent?: React.ReactNode;
   suffixComponent?: React.ReactNode;
+  isInline?: boolean;
 }
 
 /**
  * CopyToClipboard item to display customized ClipboardCopy component.
  *
  * @param prefixComponent - any component before ClipboardCopy
  * @param suffixComponent - any component after ClipboardCopy
+ * @param isInline - whether to use inline style for the component
  * @param children - the content to be used for ClipboardCopy
  */
 export const CopyToClipboard = ({
   prefixComponent,
   suffixComponent,
+  isInline,
   children,
 }: React.PropsWithChildren<ICopyToClipboardProps>) => (
   <Split hasGutter>
     {prefixComponent && <SplitItem>{prefixComponent}</SplitItem>}
     <SplitItem isFilled>
-      <ClipboardCopy removeFindDomNode isReadOnly hoverTip="Copy" clickTip="Copied">
+      <ClipboardCopy
+        removeFindDomNode
+        isReadOnly
+        hoverTip="Copy"
+        clickTip="Copied"
+        variant={isInline ? 'inline-compact' : 'inline'}
```

Comment:
When `variant = 'inline-compact'`, there is a console warning for `findDOMNode` issue. We have already put `removeFindDomNode` keyword but seems there is an issue when using `inline-compact`. We may need to report to PatternFly team to see if this can be resolved. I do doubt whether they will fix it because the last time they committed to V4 was in[ May 2021](https://github.com/patternfly/patternfly-react/commits/patternfly-patternfly-4.x).
![image](https://user-images.githubusercontent.com/6870455/224882965-017080eb-e97c-483a-8e5b-9224f70477cf.png)

Code for verification:
1. Case has warning:
```    <ClipboardCopy removeFindDomNode isReadOnly hoverTip="Copy" clickTip="Copied" variant={'inline-compact'}>
              Test Content
            </ClipboardCopy>```
2. Case without warning (variant={'inline'} is the default setting):
```    <ClipboardCopy removeFindDomNode isReadOnly hoverTip="Copy" clickTip="Copied" variant={'inline'}>
              Test Content
            </ClipboardCopy>```

Reply:
@matedo1